app/src/app/[lng]/cities/page.tsx

```diff
@@ -0,0 +1,228 @@
+      <Link href="/" _hover={{ textDecoration: "none" }}>
+        <Box
+          display="flex"
+          alignItems="center"
+          gap="8px"
+          color="content.tertiary"
+        >
+          <Text
+            textTransform="capitalize"
+            fontFamily="heading"
+            fontSize="body.lg"
+            fontWeight="normal"
+          >
+            {t("go-back")}
+          </Text>
+        </Box>
+      </Link>
+      <Box display="flex" alignItems="center" justifyContent="space-between">
+        <Heading
+          fontSize="headline.lg"
+          fontWeight="semibold"
+          color="content.primary"
+          mb={12}
+          mt={2}
+          className="w-full"
+        >
+          {t("cities-heading")}
+        </Heading>
+        {selectedRowKeys.length > 0 && (
+          <Box
+            flex={1}
+            display="flex"
+            w="full"
+            alignItems="center"
+            gap={2}
+            justifyContent="space-between"
+          >
+            <Text>
+              {t("selected-cities", {
+                count: selectedRowKeys.length,
+              })}
+            </Text>
+            <Button
+              onClick={() => {
+                setIsModalOpen(true);
+              }}
+              variant="outline"
+              h="48px"
+              minWidth="120px"
+              mt="auto"
+            >
+              {t("move")}
+            </Button>
+          </Box>
+        )}
+      </Box>
+      <Box>
+        {isLoading && <ProgressLoader />}
+        {!isLoading && (data?.length === 0 || !data) && (
+          <Text color="content.tertiary" fontSize="body.lg">
+            {t("no-data")}
+          </Text>
+        )}
+        {data && data.length > 0 && (
+          <DataTable
+            data={data}
+            searchable
+            pagination
+            itemsPerPage={20}
+            t={t}
+            title={t("manage-cities-heading")}
+            columns={[
+              { header: t("city"), accessor: "name" },
+              {
+                header: t("country"),
+                accessor: "country",
+              },
+              { header: t("organization"), accessor: null },
+              { header: t("email"), accessor: null },
+              { header: "", accessor: null },
+            ]}
+            selectable
+            selectKey="cityId"
+            selectedRowKeys={selectedRowKeys}
+            onSelectRow={(selectedRows: any[]) => {
+              setSelectedRowKeys(selectedRows);
+            }}
+            renderRow={(item, idx) => (
+              <Table.Row key={idx}>
+                <Table.Cell>{item.name}</Table.Cell>
+                <Table.Cell>{item.country}</Table.Cell>
+                <Table.Cell>
+                  {item?.project?.organization?.name ?? "N/A"}
+                </Table.Cell>
+                <Table.Cell>
+                  {item?.project?.organization?.contactEmail ?? "N/A"}
```

Comment:
I think `N/A` needs to be translated when using different languages, so let's use the i18n for it too